README.md

```diff
@@ -39,10 +38,25 @@ layer.addTo(map);
 map.fitBounds(layer.getBounds());
 ```
 
-# advanced usage
+The following options are supported:
+
+| **Option**                   | **Data Type** | **Default value**                          | **Description**                                                                                                                                                                                     |
+|------------------------------|---------------|--------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| baseUrl                      | string        | `href` of the `self` link in the STAC data | The base URL for relative links. Should be provided if the STAC data has no self link and links are not absolute. |
+| buildUrlTemplate             | function      | undefined                                  | For server-side rendering of imagery. See the chapter [using a tiler](#using-a-tiler) and [buildTileUrlTemplate](#buildTileUrlTemplate) for details. |
+| displayPreview               | boolean       | false                                      | Allow to display the asset with role `thumbnail` or the link with relation type `preview`. The previews are usually not covering the full extents and as such may be placed incorrectly on the map. |
+| displayOverview              | boolean       | true                                       | Allow to display the asset with role `overview` or `visual`. |
+| debugLevel                   | integer       | 0                                          | The higher the value the more debugging messages will be logged to the console. `0` to disable logging. |
+| latLngBounds / bounds / bbox | array<number> | undefined                                  | For STAC Assets only: The bounding box of the asset |
+| resolution                   | integer       | 32                                         | Adjust the display resolution, a power of two such as 32, 64, 128 or 256. By default the value is set to render quickly, but with limited resolution. Increase this value for better resolution, but slower rendering speed (e.g., 128). |
+| tileUrlTemplate              | string        | undefined                                  | For server-side rendering of imagery. See the chapter [using a tiler](#using-a-tiler) and [tileUrlTemplate](#tileUrlTemplate) for details. |
+| titiler                      | string        | undefined                                  | URL to a titiler instance that can be used to serve tiles instead of rendering the imagery client-side. |
```

Comment:
The titiler option was removed with the removal of the non-standard compositing of separated R-G-B bands. 
 People can still use titiler though through the other options:
```js
tileUrlTemplate: "https://titiler.tld/cog/tiles/{z}/{x}/{y}?url={url}"
```

Reply:
Some test examples are here: https://github.com/stac-utils/stac-layer/tree/main/src/test/tilers/titiler.  Wondering if we want to include links to the tests here?  Or create separate examples?

Reply:
Interesting, I found it in the code. So are the following occurrences outdated and should be removed?

- https://github.com/stac-utils/stac-layer/blob/1e2ffb7db2c91ee7b37404edf0c715ffd880a3d6/src/test/tilers/titiler/cbers.html#L33
- https://github.com/stac-utils/stac-layer/blob/ed80a5eb498d228318adfbf7fc089a6d812e7a57/src/index.js#L91-L92

Reply:
Good catch.  I'll remove those.

Reply:
Done with https://github.com/stac-utils/stac-layer/commit/041461c9ec73d93bec2cb25b5fe36b974954c85e

Reply:
I'm wondering what you would like me to do with these two files?
https://github.com/stac-utils/stac-layer/blob/main/src/utils/titiler.js and https://github.com/stac-utils/stac-layer/blob/main/src/utils/titiler-layer.js.  They are both not being used, but could be very helpful in the future if you decide to add functionality to this library that specifically requires ti-tiler.

Reply:
Or we could even pull these out into a separate NPM package if you like?

Reply:
Just leave them in for now, as you said they could be useful for some. Maybe we should add a short readme for it?

Reply:
sounds good